var/spack/repos/builtin/packages/icarus/package.py

```diff
@@ -38,3 +41,14 @@ def create_install_folders(self):
         mkdirp(join_path(prefix.lib, "ivl", "include"))
         mkdirp(join_path(prefix.share, "man"))
         mkdirp(join_path(prefix.share, "man", "man1"))
+
+    # We need to fix the CC and CXX paths, as they point to the spack
+    # wrapper scripts which aren't usable without spack
+    @run_after("install")
+    def patch_compiler(self):
+        filter_file(
+            r"^CC\s*=.*", f"CC={self.compiler.cc}", join_path(self.prefix.bin, "iverilog-vpi")
+        )
+        filter_file(
+            r"^CXX\s*=.*", f"CXX={self.compiler.cxx}", join_path(self.prefix.bin, "iverilog-vpi")
+        )
```

Comment:
```suggestion
    filter_compiler_wrappers("iverilog-vpi", relative_root="bin")
```

should be a shorthand for this

Reply:
OK that is arguably better, I think i will change that actually and there is another package that I found it in I might also fix up